tornjak-frontend/src/components/spiffe-helper.js

```diff
@@ -122,9 +122,10 @@ class SpiffeHelper extends Component {
   // performance is impacted.
   getAgentsEntries(agents, entries) {
     let nodeEntries = [];
-    if (entries !== undefined) {
-      nodeEntries = entries.filter(e => e.parent_id.path === "/spire/server");
+    if (entries === undefined) {
```

Comment:
undefined comparison not needed, also return could be empty. Alternative could be:
```
    if (!entries) {
      return;
    }
```

Reply:
File not of concern on this PR. Will be addressed in the next consecutive PR.